views/cellMove/selectReception.njk

```diff
@@ -0,0 +1,161 @@
+        </div>
+      {% endif %}
+    </div>
+  </td>
+  <td class="{{ cellClass }}">
+    <p class="govuk-body govuk-!-margin-bottom-1">
+      {{ occupier.csra }}
+    </p>
+
+    <div>
+      <a href="{{ occupier.csraDetailsUrl }}" class="govuk-link" data-test="csra-link">View details <span class="govuk-visually-hidden">for {{ occupier.name }} CSRA</span></a>
+    </div>
+  </td>
+  <td class="{{ cellClass }}">
+    {% if occupier.alerts.length %}
+      {{ alertFlags(occupier.alerts) }}
+    {% else %}
+      <p class="govuk-body govuk-!-margin-bottom-1">None</p>
+    {% endif %}
+  </td>
+  {% else %}
+    <td class="{{ cellClass }}">
+    </td>
+    <td class="{{ cellClass }}">
+    </td>
+    <td class="{{ cellClass }}">
+    </td>
+  {% endif %}
+
+  {% if index === 1 %}
+     <td class="{{ cellClass }}"><a href="{{ '/prisoner/' + offenderNo + '/cell-move/consider-risks?cellId=' + cell.id }}" class="govuk-link">Select reception</a></td>
+  {% else %}
+      <td class="{{ cellClass }}">
+      </td>
+  {% endif %}
+
+{% endmacro %}
+
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    items: [
+      {
+        text: "Home",
+        href: '/'
+      },
+      {
+        text: breadcrumbPrisonerName,
+        href: '/prisoner/' + offenderNo
+      },
+      {
+        text: 'Select Reception'
+      }
+    ]
+}) }}
+{% endblock %}
+
+{% block content %}
+  <h1 class="govuk-heading-l">{{title}}</h1>
+
+  {% include "./partials/prisonerInformation.njk" %}
+
+
+  {% if showNonAssociationWarning %}
+    {{ govukWarningText({
+      attributes: {
+        id: 'non-association-warning'
+      },
+      text: breadcrumbPrisonerName + " has a non-association with a prisoner in this location.",
+      iconFallbackText: "Warning"
+    }) }}
+  {% endif %}
+
+  {% if cells.length %}
+    <table class="govuk-table govuk-table--with-child-rows" data-test="cell-results-table">
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header" data-test="location-table-header">Location</th>
+          <th scope="col" class="govuk-table__header">Cell type</th>
+          <th scope="col" class="govuk-table__header">Capacity</th>
+          <th scope="col" class="govuk-table__header">Spaces</th>
+          <th scope="col" class="govuk-table__header">Occupier</th>
+
+          <th scope="col" class="govuk-table__header">CSRA</th>
+          <th scope="col" class="govuk-table__header">Relevant alerts</th>
+          <th scope="col" class="govuk-table__header"><span class="govuk-visually-hidden">Select cell</span></th>
+        </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+        {% for cell in cells %}
+          {% if cell.occupants.length > 0 %}
+              {% for occupier in cell.occupants %}
+                <tr class="govuk-table__row">
+                  {{ makeRow(cell, loop.index, cell.occupants.length === loop.index, occupier) }}
```

Comment:
there is a loop.last boolean you could use

Reply:
at this stage, the display of reception on this page was merely a proof of concept. So it's not really necessary to change the logic too much when compared with the nunjucks template this page was cloned from.

Reply:
not sure what you mean, i was just suggesting changing 'cell.occupants.length === loop.index' to the simpler but identical 'loop.last'

Reply:
oh I see